### PR TITLE
[zh-cn] Update response.url

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

clear untranslated content

### Related issues and pull requests

part of #14272 

blocked by https://github.com/mdn/content/pull/28412

### CODE REVIEW

File: files/zh-cn/web/api/response/url/index.md [hunk 1]
```diff
@@ -1,52 +1,46 @@
...
-  console.log(response.url); // returns https://developer.mozilla.org/zh-CN/docs/Web/API/Response/flowers.jpg
-  response.blob().then(function (myBlob) {
-    var objectURL = URL.createObjectURL(myBlob);
+fetch(myRequest).then((response) => {
+  console.log(response.url); // 返回 https://developer.mozilla.org/en-US/docs/Web/API/Response/flowers.jpg
+  response.blob().then((myBlob) => {
+    const objectURL = URL.createObjectURL(myBlob);
     myImage.src = objectURL;
   });
 });
```
Comment (yin1999) [inline files/zh-cn/web/api/response/url/index.md:31]: This code example is not match with [the source file](https://github.com/mdn/dom-examples/blob/main/fetch/fetch-response/index.html). Could you also update the en-US document :)

File: files/zh-cn/web/api/response/url/index.md [hunk 2]
```diff
@@ -1,52 +1,46 @@
...
+## 浏览器兼容性
 
 {{Compat}}
 
-## See also
+## 参见
 
-- [ServiceWorker API](/zh-CN/docs/Web/API/ServiceWorker_API)
-- [HTTP access control (CORS)](/zh-CN/docs/Web/HTTP/Access_control_CORS)
+- [ServiceWorker API](/zh=CM/docs/Web/API/Service_Worker_API)
```
Comment (yin1999) [inline files/zh-cn/web/api/response/url/index.md:44]:
```suggestion
- [ServiceWorker API](/zh-CN/docs/Web/API/Service_Worker_API)
```

**Comment by yin1999** [comment]
I haven't notice that the file has been removed in #15095. Sorry about that

**Comment by jasonren0403** [comment]
> I haven't notice that the file has been removed in #15095. Sorry about that

Should I reopen this PR 🤔

**Comment by yin1999** [comment]
> > I haven't notice that the file has been removed in #15095. Sorry about that
> 
> Should I reopen this PR 🤔 

Merge main branch and keep this file when resolving the conflict.

**Review by yin1999 — APPROVED** [review]
Thank you @jasonren0403

**Pull request merged**